src/leiningen/new/cryogen/themes/blue/html/base.html

```diff
@@ -95,5 +95,7 @@
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 {% script "js/highlight.pack.js" %}
 <script>hljs.initHighlightingOnLoad();</script>
+{% if post.klipse %} {{post.klipse|safe}} {% endif %}
+{% if page.klipse %} {{page.klipse|safe}} {% endif %}
```

Comment:
I didn't refresh this page so I didn't see your message, and that you merged my commits. Hehe. It wasn't a lot of work in any case, enlive is really nice. I'll open a new PR.